src/Microsoft.Diagnostics.Monitoring.EventPipe/GCDump/External/Graph.cs

```diff
@@ -647,7 +647,7 @@ public void FromStream(Deserializer deserializer)
         private int m_expectedNodeCount;                // Initial guess at graph Size. 
         private long m_totalSize;                       // Total Size of all the nodes in the graph.  
         internal int m_totalRefs;                       // Total Number of references in the graph
-        internal GrowableArray<TypeInfo> m_types;       // We expect only thousands of these
+        public GrowableArray<TypeInfo> m_types;       // We expect only thousands of these
```

Comment:
This was a result of having a public gcdump pipeline with a Graph as an output. 

- In the short term, I will revert this back to internal and change all the pipelines to internal until they undergo API review.
- In the long term the pipeline will be public, so we'll have to figure out if Graph is the appropriate output structure for this pipeline. One option is to directly serialize the graph to a stream, but this would limit the usability of the output.

Reply:
Yes, although we would have to work out a better contract for the pipeline if we ever want to make it public.